runtime/jvmti/jvmtiTimers.c

```diff
@@ -208,7 +208,8 @@ jvmtiGetAvailableProcessors(jvmtiEnv* env,
 
 	ENSURE_NON_NULL(processor_count_ptr);
 
-	cpuCount = j9sysinfo_get_number_CPUs_by_type(J9PORT_CPU_ONLINE);
+	/* This implementation should be kept consistent with JVM_ActiveProcessorCount */
+	cpuCount = j9sysinfo_get_number_CPUs_by_type(J9PORT_CPU_TARGET);
 	*processor_count_ptr = ((cpuCount == 0) ? 1 : (jint) cpuCount);
```

Comment:
To make the two consistent, can you modify the `((cpuCount == 0)` to be `((cpuCount < 1)`